<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BurstLengthDialog</class>
 <widget class="QDialog" name="BurstLengthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="longBurstLengthLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Long Burst Length:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="longBurstLengthInput">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>40</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="shortBurstLengthLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Short Burst Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="shortBurstLengthMeasurementUnit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>round(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="singleShotTimeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Single Shot Time:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="singleShotTimeInput">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>85</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="singleShotTimeMeasurementUnit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>85</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>330</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="shortBurstLengthInput">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>10</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="longBurstLengthMeasurementUnit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>round(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
